cameo/strain_design/heuristic/evolutionary_based.py

```diff
@@ -478,17 +478,17 @@ def _process_solutions(self):
             self._processed_solutions = processed_solutions
 
     def display_on_map(self, index=0, map_name=None, palette="YlGnBu"):
-        with TimeMachine() as tm:
+        with self._model:
             for ko in self.data_frame.loc[index, "reactions"]:
-                self._model.reactions.get_by_id(ko).swap_cofactors(self._swap_pairs, tm)
+                self._model.reactions.get_by_id(ko).swap_cofactors(self._swap_pairs)
```

Comment:
Reaction.swap_cofactors do not exist anymore. You can use `cameo.core.manipulation.swap_cofactors`.